### DISCUSSION TITLE
Apple Music Playlist/tracks not syncing or showing

### DISCUSSION THREAD

**Discussion by Kebo2r** [music-assistant/support#2587] [Q&A] [answered]
This is a great add-on!  I just got setup and while my albums and artists show up from the Apple Music integration, the playlist and tracks are not showing at all.  I have deleted and readded Apple Music.  Any other ideas?  Thank you!

**Comment by OzGav** [comment]
There are some open issues related to this that are currently being worked on. Have a look at those and see if they match your problem. Make sure you are on beta 13. If you aren't using the beta then you will have to wait for a while

https://github.com/music-assistant/hass-music-assistant/issues/2563

**Comment by Kebo2r** [comment, marked as answer]
All is well on the latest beta on this front!  Thank you!  one more issue I will open but this integration is great!
